apps/penxle.com/src/routes/editor/Header.svelte

```diff
@@ -287,61 +402,78 @@
       {/if}
     </div>
 
-    <Tooltip>
-      <div class="flex flex-col text-right w-36.5">
-        <span class="body-15-b">
-          <mark class="text-blue-50">80</mark>
-          자
-        </span>
-        <span class="caption-12-m text-disabled">마지막으로 저장된 시간 13:00</span>
-      </div>
-
-      <div slot="message">
-        <div class="caption-12-m">
-          <span class="text-secondary w-17.5 inline-block">공백 미포함</span>
-          <span class="text-rights">80자</span>
-        </div>
-        <div class="caption-12-m">
-          <span class="text-secondary w-17.5 inline-block">공백 포함</span>
-          <span class="text-rights">80자</span>
-        </div>
-      </div>
-    </Tooltip>
+    <div class="flex flex-col items-end text-right w-36.5">
+      <ArticleCharacterCount {editor} />
+      <span class="caption-12-m text-disabled">
+        {#if revisedAt}
+          {dayjs(revisedAt).formatAsTime()} 저장됨
+        {:else}
+          저장되지 않음
+        {/if}
+      </span>
+    </div>
 
     <div class="flex">
-      <Button
-        class="body-15-b disabled:border! border-r-none rounded-r-none peer"
-        color="tertiary"
-        disabled={!canPublish}
-        loading={$revisePost.inflight}
-        size="lg"
-        variant="outlined"
-        on:click={() => revise('MANUAL_SAVE')}
-      >
-        임시저장
-      </Button>
+      <Tooltip enabled={!canRevise} message={reviseNotAvailableReason}>
+        <Button
+          class="body-15-b disabled:border! border-r-none rounded-r-none peer"
+          color="tertiary"
+          disabled={!canRevise}
+          loading={$revisePost.inflight}
+          size="lg"
+          variant="outlined"
+          on:click={() => doRevisePost('MANUAL_SAVE')}
+        >
+          임시저장
+        </Button>
+      </Tooltip>
+
       <Button
         class="text-blue-50 body-15-b rounded-l-none peer-enabled:peer-hover:border-l-border-primary"
```

Comment:
P1: 버튼이 툴팁 안에 들어가면서 임시저장 버튼 스타일이 적용이 안되는 것 같아요! 이렇게 수정해주시면 될 것 같습니다.

```suggestion
      <Tooltip class="peer" enabled={!canRevise} message={reviseNotAvailableReason}>
        <Button
          class="body-15-b disabled:border! border-r-none rounded-r-none"
          color="tertiary"
          disabled={!canRevise}
          loading={$revisePost.inflight}
          size="lg"
          variant="outlined"
          on:click={() => doRevisePost('MANUAL_SAVE')}
        >
          임시저장
        </Button>
      </Tooltip>

      <Button
        class="text-blue-50 body-15-b rounded-l-none peer-[&:has(button:enabled)]:peer-hover:border-l-border-primary"
```